### PR TITLE
Changes to README.md

### PR DESCRIPTION
**Pull request by utkarsh-rg**
I have done some changes in README.md which I think you should merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # BomberMenAI
 PvP Bomberman game
+This is a game that someone else created.
```
Comment (Twins-Divyanshu-Sharma) [inline README.md]: This is bullshit.Remove this line

**Review by Twins-Divyanshu-Sharma — CHANGES_REQUESTED** [review]

**Review by Twins-Divyanshu-Sharma — APPROVED** [review]
Looks goo

**Comment by Twins-Divyanshu-Sharma** [comment]
There is a conflict please resolve that. Will merge after the conflict is resolved.

**Comment by Twins-Diwakar-Sharma** [comment]
OK thank you, utkarsh-rg

**Pull request merged**